src/EditorFeatures/CSharpTest/NavigateTo/NavigateToTests.cs

```diff
@@ -1068,9 +1090,8 @@ public void ToError(__arglist)
 }", async w =>
 {
     var item = (await _aggregator.GetItemsAsync("ToError")).Single();
-    VerifyNavigateToResultItem(item, "ToError", "[|ToError|](__arglist)", MatchKind.Exact, NavigateToItemKind.Method, Glyph.MethodPublic);
+    VerifyNavigateToResultItem(item, "ToError", "[|ToError|](__arglist)", PatternMatchKind.Exact, NavigateToItemKind.Method, Glyph.MethodPublic);
```

Comment:
I haven't looked at all the test changes.  Can i assume there is no observable change in behavior?

Reply:
@CyrusNajmabadi , most test changes were straightforward, but TermSplittingTests towards the bottom are now mapped to the new kinds of CamelCase.  Would love it if you could take a look to see if I've mapped those correctly.

For example, when searching for "GKW" is CamelCaseNonContiguousPrefix the correct match for "get_key_word"?  And when searching for "K W" or "K*W" do we expect "get_key_word" is CamelCaseSubstring (not case sensitive) and "GetKeyWord" is SubstringPatternMatch?

I did some adhoc testing as well and the new order of matches is looking really good.

Reply:
I verified this is not a regression. Searching for GKW matches get_key_word with CamelCaseNonContiguousPrefix kind even before your changes.